### DISCUSSION TITLE
How to disable netdata updater?

### DISCUSSION THREAD

**Discussion by jrca025** [netdata/netdata#13073] [Q&A] [answered]
I want to disable the netdata updater or make it a weekly checker instead of checking updates on a daily basis.

**Comment by ilyam8** [comment, marked as answer]
Hi, @jrca025. You can disable updates:

- by executing `netdata-updater.sh --disable-auto-updates` after installation.
- adding `--no-updates` parameter to `kickstart.sh` for new installs.

---

> a weekly checker instead of checking updates on a daily basis

cc @netdata/agent-sre

**Reply by Ferroin** [reply]
On the note of the possibility of a weekly check instead of a daily check, that actually would make sense for users on the stable release channel, though probably not as much for nightlies, and support for that should not be too hard to add.

I’ll go ahead and create a feature request for it.

**Comment by ReifiedException** [comment]
Or you can remove the link to updater script from crontab:

`% sudo rm /etc/cron.daily/netdata-updater`

It's just a link, so you can re-create it if you change your mind later:

`lrwxrwxrwx 1 root root 39 May 27 10:38 /etc/cron.daily/netdata-updater -> /usr/libexec/netdata/netdata-updater.sh`
